config/locales/ru.yml

```diff
@@ -1096,7 +1097,8 @@ ru:
     reference: "Ссылка"
     refund: "Возврат"
     refund_amount_must_be_greater_than_zero:
-    refund_reasons: "Причины возврата"
+    refund_reasons: "Причины возврата денег"
+    return_reasons: "Причины возврата товара"
```

Comment:
Maybe my github has an old cache, but it's not in the correct alphabetical order.

As we use the original `en.yml` from solidus core as reference, would you please add translation key at the same position as in the original one?

https://github.com/solidusio/solidus/blob/master/core/config/locales/en.yml#L1299

Thanks for your understanding and your contribution :tulip: